lib/tty/box.rb

```diff
@@ -240,6 +241,8 @@ def frame(*content, top: nil, left: nil, width: nil, height: nil, align: :left,
           size = Strings::ANSI.sanitize(content[i]).scan(/[[:print:]]/).join.size
           content_size -= size
         end
+        # handle occasional negative numbers
+        content_size = [content_size, 0].max
```

Comment:
Could you please add a test? These kind of things are important to catch in a test case to prevent future breakages due to refactorings etc.